src/api/WriteOutputRequest.java

```diff
@@ -0,0 +1,5 @@
+package api;
+
+public class WriteOutputRequest implements WriteOutputResponse {

```

Comment:
WriteOutputRequest is not a type of WriteOutputResponse, and shouldn't be implementing this interface.